features/models.py

```diff
@@ -68,6 +68,12 @@ class Feature(TranslatableModel, TimestampedModel):
         url=models.URLField(
             verbose_name=_("url"), blank=True, help_text=_("URL of the feature")
         ),
+        one_liner=models.CharField(
+            verbose_name=_("one-liner"),
+            max_length=256,
+            blank=True,
+            help_text=_("One-liner, length limited to 256 chars"),
```

Comment:
Since one-liner is the name of the field already the help text could be something in the lines of: "A short description, limited to 256 chars"